src/file-1.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:35583/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:35583/project/issues?id=sample&pullRequest=5504&issues=AWcX7ZXu_FG9yBfilam_&open=AWcX7ZXu_FG9yBfilam_)

[ISSUE_KEY]: AWcX7ZXu_FG9yBfilam_